buildtest/cli/help.py

```diff
@@ -492,6 +492,20 @@ def print_unittests_help():
     console.print(table)
 
 
+def print_tutorialexamples_help():
+    """This method will print help message for command ``buildtest help tutorial-examples``"""
+
+    table = Table(title="Generate tuotial examples", show_lines=False)
+    table.add_column("Command", justify="left", style="cyan")
+    table.add_column("Description", justify="left", style="magenta")
+
+    table.add_row(
+        "buildtest tutorial-examples",
+        "Generate documentation examples for Buildtest Tutorial",
+    )
+    console.print(table)
+
+
```

Comment:
i think for now we can remove this section, i realize we don't document every single command such as `buildtest docs` since `buildtest tutorial-examples` is just another command that runs without any arguments we could just remove this for now

Reply:
@shahzebsiddiqui Okay I have removed this part.